server/app/src/main/java/com/ams/app/dto/Statsresponsedto.java

```diff
@@ -0,0 +1,16 @@
+package com.ams.app.dto;
+
+import lombok.Data;
+
+@Data
+public class Statsresponsedto {
+    private String subjectname;
+    private float percentage;
+    private int subid;
+    
+    public Statsresponsedto(String subjectname, float percentage, int subid) {
```

Comment:
Why do we need this? @Data will have a AllArgsConstructor by default. If not, Use @AllArgsConstructor

Reply:
Used @AllArgsConstructor